doc/classes/EditorSettings.xml

```diff
@@ -465,15 +465,15 @@
 		</member>
 		<member name="interface/editor/code_font_contextual_ligatures" type="int" setter="" getter="">
 			The font ligatures to enable for the currently configured code font. Not all fonts include support for ligatures.
-			[b]Note:[/b] The default editor code font (Hack) does not have contextual ligatures in its font file.
+			[b]Note:[/b] The default editor code font ([url=https://www.jetbrains.com/lp/mono/]JetBrains Mono[/url]) has contextual ligatures in its font file.
 		</member>
 		<member name="interface/editor/code_font_custom_opentype_features" type="String" setter="" getter="">
 			List of custom OpenType features to use, if supported by the currently configured code font. Not all fonts include support for custom OpenType features. The string should follow the OpenType specification.
-			[b]Note:[/b] The default editor code font (Hack) does not have custom OpenType features in its font file.
+			[b]Note:[/b] The default editor code font ([url=https://www.jetbrains.com/lp/mono/]JetBrains Mono[/url]) has custom OpenType features in its font file, but [url=https://github.com/JetBrains/JetBrainsMono/issues/478]there is no documented list yet[/url].
```

Comment:
I'm not convinced about linking to an old issue that's not getting activity and doesn't answer the question.
If they do eventually add documentation we'll likely miss updating this link so that will become bad docs.

If users want to know which features are supported, they can go to the linked JetBrains Mono website and find the relevant community platform there to nag about it.